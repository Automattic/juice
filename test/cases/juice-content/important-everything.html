<html>
<head>
    <style>
      a { font-family: bold }
    </style>
</head>
<body>
<a>hello</a>
</body>
</html>
