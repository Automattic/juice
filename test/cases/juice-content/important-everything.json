{
    "url": "./",
    "removeStyleTags": true,
    "importantEverything": true
}
