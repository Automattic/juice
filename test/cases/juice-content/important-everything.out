<html>
<head>
    
</head>
<body>
<a style="font-family: bold !important;">hello</a>
</body>
</html>
